Sources/Brave/Frontend/Browser/TabManager.swift

```diff
@@ -87,8 +87,8 @@ class TabManager: NSObject {
   var tempTabs: [Tab]?
   private weak var rewards: BraveRewards?
   private weak var tabGeneratorAPI: BraveTabGeneratorAPI?
-  var makeWalletEthProvider: ((Tab) -> (BraveWalletEthereumProvider, js: String)?)?
-  var makeWalletSolProvider: ((Tab) -> (BraveWalletSolanaProvider, jsScripts: [BraveWalletProviderScriptKey: String])?)?
+//  var makeWalletEthJS: (() -> String?)?
+//  var makeWalletSolJSScripts: (() -> [BraveWalletProviderScriptKey: String]?)?
   private var domainFrc = Domain.frc()
```

Comment:
can be removed.

Reply:
Thanks, cleaned up in [0702fa0](https://github.com/brave/brave-ios/pull/7483/commits/0702fa059ca6593d4e312ab0d5b2e0e250a031ac)